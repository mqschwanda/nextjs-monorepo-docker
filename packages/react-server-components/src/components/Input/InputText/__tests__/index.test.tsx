import { render } from '@testing-library/react';
import { InputText } from '@';

describe('@mqs/react-server-components', () => {
  describe('components', () => {
    describe('<InputText />', () => {
      it('renders', () => {
        const testId = InputText.name;

        const { getByTestId, asFragment } = render((
          <InputText
            className='test'
            testId={testId}
          />
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
