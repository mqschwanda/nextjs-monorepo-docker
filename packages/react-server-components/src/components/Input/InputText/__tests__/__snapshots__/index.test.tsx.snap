// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-server-components components <InputText /> renders 1`] = `
<DocumentFragment>
  <input
    class="input test"
    data-testid="InputText"
  />
</DocumentFragment>
`;
